build/KoreBuild.sh

```diff
@@ -152,7 +152,13 @@ export ReferenceAssemblyRoot=$NUGET_PACKAGES/netframeworkreferenceassemblies/$ne
 makeFileProj="$scriptRoot/KoreBuild.proj"
 msbuildArtifactsDir="$repoFolder/artifacts/msbuild"
 msbuildResponseFile="$msbuildArtifactsDir/msbuild.rsp"
-msbuildLogFile="$msbuildArtifactsDir/msbuild.binlog"
+msBuildLogArgument=""
+
+if [ ! -z "$KOREBUILD_ENABLE_BINARY_LOG" ]; then
```

Comment:
Feel free to file an issue (or even do clean up) as neither I know what is best practice in scripting languages nor I know practice followed in this repo around this.

I created this PR simply to unblock EF on travis.